src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import Form from "./components/Form";
+import TodoList from "./components/TodoList";
+
+function App() {
+  const [inputText, setInputText] = useState("");
+  const [todos, setTodos] = useState([]);
+  const [status, setStatus] = useState("all");
+  const [filteredTodos, setFilteredTodos] = useState([]);
+
+  useEffect(() => {
+    getLocalTodos();
+  }, []);
+
+  useEffect(() => {
+    filterHandler();
+    SaveLocalTodos();
+  }, [todos, status]);
```

Comment:
https://stackoverflow.com/questions/55840294/how-to-fix-missing-dependency-warning-when-using-useeffect-react-hook

<img width="723" alt="Zrzut ekranu 2023-01-27 o 17 26 25" src="https://user-images.githubusercontent.com/56917808/215139377-21e8d552-7b91-415a-af73-28ab811093a6.png">
Można taki komentarz dodać i w tedy nie będzie tego błędu tutaj